src/number_theory/padics/padic_integers.lean

```diff
@@ -11,7 +11,7 @@ import topology.metric_space.cau_seq_filter
 /-!
 # p-adic integers
 
-This file defines the p-adic integers `ℤ_p` as the subtype of `ℚ_p` with norm `≤ 1`.
+This file defines the `p`-adic integers `ℤ_p` as the subtype of `ℚ_p` with norm `≤ 1`.
```

Comment:
The way I see it, the general criterion for using backticks is that you have to be writing down Lean code. If you're using more general math notation, then you shouldn't have backticks. 

Under this criterion, writing down `` `p`-adic`` instead of `p-adic` is fine, although not necessary. Writing `` `ℤ_p` `` or `` `ℤ[p]` `` isn't, since the Lean notation is `ℤ_[p]`.

Perhaps this is something we should formalize in the mathlib style guide.

Reply:
Probably math notation should be written either in `$$` or as lean code in ``, rather than a weird in-between.

Reply:
I'll switch everything to Lean code for this.